src/generateDiff.ts

```diff
@@ -1,12 +1,14 @@
-const diff = require('diff');
+import * as diff from 'diff';
+
+export type Change = diff.Change & { noNewline?: boolean };
```

Comment:
Does this need to be exported? :thinking: I think it's strictly internal.

```suggestion
type Change = diff.Change & { noNewline?: boolean };
```